lib/datapacksutils.js

```diff
@@ -4917,4 +4917,52 @@ var SALESFORCE_FOLDER_TO_TYPE =  {
     triggers: "ApexTrigger",
     weblinks: "CustomPageWebLink",
     workflows: "Workflow"
-}	
\ No newline at end of file
+}
+
+DataPacksUtils.prototype.updateDataPackWithNewMappings = function (dataPack, jobInfo) {
+    let obj = {};
+    let dataPackType = dataPack.VlocityDataPackType;
+
+    let dataField = this.getDataField(dataPack);
+    let mappings = jobInfo.platformObjectMappings[dataPackType][dataField];
+
+    if (mappings) {
+        let newDataField = mappings.objectName;
+        obj[newDataField] = dataPack.VlocityDataPackData[dataField];
+
+        //Iterate through keys of <dataPack.VlocityDataPackData[dataField]> to update the keys(fieldNames)
+        Object.keys(obj[newDataField][0]).forEach(key => {
```

Comment:
In this you are missing replacing the VlocityMatchingRecordSourceKey for Element matching keys. It still shows OmniScript__c in the data for the Matching Key.

Reply:
There are a few other fields that it needs to be replaced in. If you convert an OmniScript then search through the file for OmniScript__c then it should show all. I just did a find and replace and then it deployed correctly.